### PR TITLE
Filter logs (Closes #5)

### LINKED ISSUE #5: Filter lines by a regex

### PR DESCRIPTION
**Pull request by sankha93**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -27,10 +27,20 @@
       padding: 5px;
       width: 100%;
     }
+    .filter {
+      float: right;
+      padding-right: 10px;
```
Comment (dash1291) [inline static/index.html]: this is flowing out of the window, try `margin-right: 20px`

**Comment by dash1291** [comment]
i'm also blank lines probably for the lines which don't match. please fix that and i'll merge this

**Comment by sankha93** [comment]
Updated.

**Comment by dash1291** [comment]
Thank you for your contribution!

**Pull request merged**